WasatchNET/AndorEEPROMJSON.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace WasatchNET
+{
+    public class AndorEEPROMJSON
+    {
+        public string detector_serial_number;
+        public string detector_type;
+        public double[] wavelength_coeffs;
+        public double excitation_nm_float;
+        public string wp_serial_number;
+        public string wp_model;
+        public bool invert_x_axis;
+        public double[] raman_intensity_coeffs;
```

Comment:
Do we need roi_horizontal_start and end? I think those are required for use with raman_intensity_coeffs.

Reply:
I don't think we use this file with anything (there's a duplicate in WPP...for some reason)